src/cache_invalidate.c

```diff
@@ -59,45 +60,44 @@ cache_invalidate_relcache_all(void)
 	ts_bgw_job_cache_invalidate_callback();
 }
 
+static Oid hypertable_proxy_table_oid = InvalidOid;
+static Oid bgw_proxy_table_oid = InvalidOid;
+
+void
+ts_cache_invalidate_set_proxy_tables(Oid hypertable_proxy_oid, Oid bgw_proxy_oid)
+{
+	hypertable_proxy_table_oid = hypertable_proxy_oid;
+	bgw_proxy_table_oid = bgw_proxy_oid;
+}
```

Comment:
This is not used anywhere else than in this file, so why does it need external linkage?

Reply:
It is called from `catalog.c` when the `Catalog` object is initialized it sets these to valid table relids.

Reply:
Missed that one. All good.